src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppgaverForBarnService.kt

```diff
@@ -26,10 +26,10 @@ class OpprettOppgaverForBarnService(private val oppgaveClient: OppgaveClient,
         oppgaverForBarn.forEach {
             try {
                 taskRepository.save(Task(OpprettOppgaveForBarnTask.TYPE,
-                                         objectMapper.writeValueAsString(it),
-                                         Properties()))
+                                         objectMapper.writeValueAsString(it)))
             } catch (ex: Exception) {
                 logger.error("Kunne ikke opprette task for barn som fyller år med OppgaveForBarn=$it")
```

Comment:
Ikke koblet til PR, men `it` her logger hele barnobjektet i vanlig logg, er det ønskelig?

Reply:
Hvis vi skal logge `it` så tenker jeg at `securelogger` er riktig nivå.

Reply:
Ja, bra, legger den i securelogger i stedet